webpack/envs/clientProductionConfig.js

```diff
@@ -25,15 +25,14 @@ export const clientProductionConfig = {
       seed: env.isProduction ? getCSSManifest() : {},
     }),
   ],
-  optimization: env.webpackDebug
-    ? {}
-    : {
-        minimizer: [
-          new TerserPlugin({
-            cache: false,
-            parallel: false,
-            sourceMap: true, // Must be set to true if using source-maps in production
-          }),
-        ],
-      },
+  optimization: {
+    minimize: !env.webpackDebug,
+    minimizer: [
+      new TerserPlugin({
+        cache: false,
+        parallel: env.onCi ? 4 : true, // Only use 4 cpus in CircleCI, by default it will try using 36 and OOM
```

Comment:
**suggestion (non-blocking):** Use an environment variable to drive the cpu count on CI

If builds start failing due to OOM errors in the  future, it'd be nice to be able to quickly update this via circle's configuration instead of having to make a code change. 

<!--
Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.
-->